### ISSUE TITLE
Any chance to translate README to English?

### ISSUE THREAD

**Issue by alkhimey** [stcarrez/bbox-ada-api#1]
This repository looks interesting, but I can not understand what this project is about.

**Comment by stcarrez** [comment]
Yes... Sorry about that but it concerns the use of an API for Bouygues Telecom gateways, so it's rather targeted at French people. I'll try to make an English section.
